tests/unit/specs_test.bzl

```diff
@@ -1,4 +1,4 @@
-load("@bazel_skylib//lib:unittest.bzl", "asserts", "unittest")
+load("@bazel_skylib_local//lib:unittest.bzl", "asserts", "unittest")
```

Comment:
Why the change to bazel_skylib_local? In the workspace you imported skylib as "io_bazel_skylib"

Reply:
That's io_bazel_skydoc, not io_bazel_skylib. skylib_repositories imports its own bazel_skylib at an another version: https://github.com/bazelbuild/skydoc/blob/f57c9880b9033241ad341ffc17a88d804cdfde3a/setup.bzl#L67

Reply:
Ah, yes, my mistake on the skylib/skydoc swap.

But I still have a comment: are the two versions of skylib incompatible (the version that's pulled in at the top level and the version that skydoc_repositories pulls in)? Because the skydoc_repositories one is inside of an "if_not_defined" block, so if the versions are compatible, the rules_jvm_external declaration of bazel_skylib can be moved on top of the call to skydoc_repositories and then there will only be one bazel_skylib

Reply:
Right, yeah but that means that I will make Stardoc depend on my version of bazel_skylib, which may or may not work. Should diamond splitting be used here instead?

Reply:
The ideal scenario is that that will work :) 

The way it is now is effectively diamond splitting so it's fine to leave it as is if you don't merge the two bazel_skylib versions

Reply:
This is now fixed.